package.json

```diff
@@ -1,27 +1,34 @@
 {
   "name": "loki-grpc-client",
   "version": "0.1.0",
-  "description": "Skeleton template for a balena TypeScript project",
-  "homepage": "https://github.com/balena-io/balena-typescript-skeleton#readme",
+  "description": "Loki GRPC client for Node.js",
+  "homepage": "https://github.com/balena-io-modules/node-loki-grpc-client#readme",
   "main": "build/index.js",
   "types": "build/index.d.ts",
   "keywords": [
     "balena",
-    "typescript"
+    "typescript",
+    "loki",
+    "grpc",
+    "logging",
+    "logs"
   ],
   "author": "",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/balena-io/balena-typescript-skeleton.git"
+    "url": "git+https://github.com/balena-io-modules/node-loki-grpc-client.git"
   },
   "bugs": {
-    "url": "https://github.com/balena-io/balena-typescript-skeleton/issues"
+    "url": "https://github.com/balena-io-modules/node-loki-grpc-client/issues"
   },
   "files": [
-    "build/"
+    "build/",
+    "proto/"
   ],
   "scripts": {
+    "protoc:compile": "npx grpc_tools_node_protoc --js_out=import_style=commonjs,binary:./proto/compiled --grpc_out=./proto/compiled --plugin=protoc-gen-grpc=`which grpc_tools_node_protoc_plugin` -I ./proto ./proto/*.proto",
+    "protoc:types": "npx grpc_tools_node_protoc --plugin=protoc-gen-ts=./node_modules/.bin/protoc-gen-ts --ts_out=./proto/compiled -I ./proto ./proto/*.proto",
```

Comment:
I think these need to be called as part of either the `prepack` or `build` scripts right?